### DISCUSSION TITLE
IIP-9: Change the epoch length

### DISCUSSION THREAD

**Discussion by midenaio** [idena-network/idena-docs#139] [IIPs]
Change the minimum epoch length to 14 days for the network that exceeds 291 identities.

Read more details [here](https://docs.idena.io/docs/iip/iip-9).

**Comment by teavar** [comment]
Many id's are in pools so their choice in this does not count because 1 pool = 1 vote. Since this proposal will affect all id's how can there be a fair voting on this proposal?

**Comment by teavar** [comment]
a 14 day interval will probably be good for most if not all participants but it could be unfair for some or maybe even many people if the validation will always fall on a Saturday

**Reply by NPC69** [reply]
Idena will probably never be available to everyone due to its nature without sacrificing its security, so I think this is preferable because:

1. Saturday is not a workday
2. Consistency

**Comment by midenaio** [comment]
@teavar thanks for your comments. 

>Many id's are in pools so their choice in this does not count because 1 pool = 1 vote. Since this proposal will affect all id's how can there be a fair voting on this proposal?

When an identity delegates mining to a pool, it loses its voting power. This is a security measure to avoid manipulating votes by pools' accounts.

>a 14 day interval will probably be good for most if not all participants but it could be unfair for some or maybe even many people if the validation will always fall on a Saturday

The choice of a validation day for Saturday is not subject to this proposal. Saturday is already [used for validations](https://docs.idena.io/docs/wp/technology#validation-session-schedule) when the network is greater than 9441
